packages/contracts-bedrock/src/libraries/GasPayingToken.sol

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
```

Comment:
This library needs some unit tests.

- Fuzz roundtrip of `set` + each `getXXX`
- Direct `assertEq` on each slot constant to its `bytes32` value